apps/netlify/frontend/src/config.js

```diff
@@ -61,9 +67,9 @@ export function parametersToConfig(parameters) {
   return {
     netlifyHookIds: readCsvParam(parameters.notificationHookIds),
     sites: buildHookIds.map((buildHookId, i) => {
-      const selectedContentTypes = parameters.events[buildHookId] === '*' ?
-      parameters.events[buildHookId] :
-      readCsvParam(parameters.events[buildHookId]);
+      const selectedContentTypes = parameters.events[buildHookId]?.cts === '*' ?
+      parameters.events[buildHookId]?.cts :
```

Comment:
Not introduced by this PR, but we crash when trying to read this from events if you have a config from the old version of the app - because the events property didn't used to exist.